src/Device/Driver/XCTracer/Parser.cpp

```diff
@@ -90,9 +90,22 @@ static int
 ReadCheckedLatitudeLongitude(NMEAInputLine &line, double &value_latitude, double &value_longitude)
 {
   int valid_fields = 0;
+  double latitude, longitude;
+  double epsilon = std::numeric_limits<double>::epsilon();
+
+  valid_fields += ReadCheckedRange(line, latitude, -90.0, 90.0);
+  valid_fields += ReadCheckedRange(line, longitude, -180.0, 180.0);
 
-  valid_fields += ReadCheckedRange(line, value_latitude, -90.0, 90.0);
-  valid_fields += ReadCheckedRange(line, value_longitude, -180.0, 180.0);
+  if(valid_fields == 2 && !(fabs(latitude) < epsilon && fabs(longitude) < epsilon)) {
```

Comment:
Not sure if this is what you expected but I wouldn't give any tolerant for this magic number.
Otherwise please advice.